MBBSEmu/HostProcess/ExportedModules/Majorbbs.cs

```diff
@@ -158,6 +157,8 @@ public void Dispose()
             Module.Memory.SetWord("PFNLVL", 0);
             Module.Memory.AllocateVariable("VERSION", 5);
             Module.Memory.SetArray("VERSION", Encoding.ASCII.GetBytes("2.00"));
+            Module.Memory.AllocateVariable("SYSTEM_NAME", 0x32, true);
+            Module.Memory.SetArray("SYSTEM_NAME", Encoding.ASCII.GetBytes(_configuration.BBSTitle));
```

Comment:
If the bbs title is 32 chars or greater it will Overflow the buffer, yes?

Limit to 31 to ensure a null terminator